### DISCUSSION TITLE
[Feature] ... Opportunity Mapped Product & Services in Quotation

### DISCUSSION THREAD

**Discussion by JEEVAN28** [YetiForceCompany/YetiForceCRM#14076] [Feature improvement]
Hello,

It will be helpful for user if Products selected or mapped in Opportunity is auto populated in Quotation. Currently user has to select products manually. This becomes cumbersome when items are more than 10 

![image](https://user-images.githubusercontent.com/12723595/105137427-48493780-5b19-11eb-8d7c-e1e3ff2c3632.png)

![image](https://user-images.githubusercontent.com/12723595/105137501-66af3300-5b19-11eb-83a6-c7aa76649226.png)

Thanks
Jeevan Pendhari

**Comment by deepti4666** [comment]
Would love this as well
